.reuse/dep5

```diff
@@ -301,43 +300,48 @@ Copyright: Icons8
 License: LicenseRef-Icons8
 
 Files: libs/delaunay-triangulation/*
-Copyright: 2015 Simon Zeni
+Copyright: 2015, Simon Zeni
 License: MIT
 
 Files: libs/dxflib/*
-Copyright: 2001 Robert J. Campbell Jr., 2001-2013 RibbonSoft GmbH
+Copyright: 2001-2013, RibbonSoft GmbH
 License: GPL-2.0-or-later
 
-Files: libs/fontobene-qt/*
-Copyright: Fontobene Developers
+Files: libs/fontobene-qt5/*
+Copyright: 2012-2020, Urban Bruhin
```

Comment:
Hmm this change makes no sense, the submodule is called `fontobene-qt`, not `fontobene-qt5` (anymore). Also I'd prefer keeping "Fontobene Developers" as copyright holder.

Reply:
Rolled back.